src/libs/ascent/hola/ascent_hola.hpp

```diff
@@ -4,22 +4,20 @@
 // other details. No copyright assignment is required to contribute to Ascent.
 //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
 
-
 //-----------------------------------------------------------------------------
 ///
 /// file: ascent_hola.hpp
 ///
 //-----------------------------------------------------------------------------
 
 #ifndef ASCENT_HOLA_HPP
-#define ASCENT_HOLA_HPP
-
-#include <ascent_config.h>
-#include <ascent_exports.h>
+#    define ASCENT_HOLA_HPP
```

Comment:
In general I like intended defs, however for the header file case there is always a indef / define that wraps the whole file to avoid multiple definitions.

It would be good to not indent the normal includes for this case.